qadi_simple.py

```
@@ -0,0 +1,342 @@
+            for i, example in enumerate(result.verification_examples):
+                # Parse example structure for better formatting
+                lines = example.split('\n')
+                
+                # Look for structured format markers
+                context_line = None
+                application_line = None
+                result_line = None
+                
+                for line in lines:
+                    line = line.strip()
+                    if line.startswith('- Context:') or line.startswith('Context:'):
```

Ruff (0.12.2)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)